### PR TITLE
Add PageData attribute changeType

### PR DESCRIPTION
**Pull request by hlbraddock**
Bulk scraper JSON specification should match real time content publisher https://github.com/CruGlobal/cru-aem-commons/blob/CRUORG-221/core/src/main/java/org/cru/commons/event/model/PageInfo.java

### CODE REVIEW

**Comment by hlbraddock** [comment]
@wrandall22 bump

File: src/main/java/org/cru/aemscraper/model/PageData.java [hunk 1]
```diff
@@ -6,6 +6,11 @@
 import java.util.Objects;
 
 public class PageData {
+
+    public enum CHANGE_TYPE {
+        Activate
```
Comment (wrandall22) [inline src/main/java/org/cru/aemscraper/model/PageData.java:11]: This is the only one because the scraper is only used for importing create/update data?

**Review by wrandall22 — APPROVED** [review]
Sorry, never got a review request for this.

File: src/main/java/org/cru/aemscraper/model/PageData.java (hunk 1 above)
Comment (hlbraddock) [reply src/main/java/org/cru/aemscraper/model/PageData.java:11]: Correct.

**Pull request merged**